builtin/game/item.lua

```diff
@@ -335,6 +335,10 @@ function core.item_place(itemstack, placer, pointed_thing, param2)
 end
 
 function core.item_drop(itemstack, dropper, pos)
+	if math.abs(pos.x) > 30910 or math.abs(pos.y) > 30910 or math.abs(pos.z) > 30910 then
+		core.log("error", "itemdrop called outside of safe map bounds! @"..core.pos_to_string(pos))
```

Comment:
The pos is actually rounded to the number as its the pos of the dug node in
that instance, thrown items i  will look into :)
On Sep 29, 2015 5:04 AM, "kilbith" <notifications@github.com> wrote:

> In builtin/game/item.lua
> <https://github.com/minetest/minetest/pull/3211#discussion_r40649061>:
>
> > @@ -335,6 +335,10 @@ function core.item_place(itemstack, placer, pointed_thing, param2)
> >  end
> >
> >  function core.item_drop(itemstack, dropper, pos)
> > +	if math.abs(pos.x) > 30910 or math.abs(pos.y) > 30910 or math.abs(pos.z) > 30910 then
> > +		core.log("error", "itemdrop called outside of safe map bounds! @"..core.pos_to_string(pos))
>
> Isn't that writing the entities raw positions with a long floating ?
> Perhaps you should use vector.round() for trim it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/minetest/minetest/pull/3211/files#r40649061>.
>